report_templates/ansible_inventory.erb

```diff
@@ -157,7 +157,7 @@ require:
       inventory_data.update('location': host.location) if input('Location') == 'yes'
       hostgroup = host.hostgroup.title if host.hostgroup
       inventory_data.update('host_group': hostgroup) if input('Host Group') == 'yes'
-      host_collections = host.host_collections.map { |c| c.name } if host.host_collections
+      host_collections = host_collections_names(host) if host.host_collections
```

Comment:
I'm not sure if this is an equivalent change, but I'm unsure how to test this template. The value is now a comma separated string of names rather than an array of names. If this change isn't required I'd rather omit it.

Reply:
I think that my addition is correct because actually we're trying to show multiple values in one cell of table. Also, this template is report template so it's intended for human, not machine (like job template) and that means that multiple value should have a delimiter.

Reply:
That's not entirely true @domitea. If I select json format as an output format, I might very likely intend to consume it by machine for further processing. That being said, we might want to include formaters for columns that would take care of different formatting of columns. what actually goes over this PR. But I believe the macro should return an array of values and templating take care of the formatting :thinking:

Reply:
I agree too, macros shouldn't handle formatting. also comma-seperated list might mess up csv reports (not sure if commas in field values are escaped when rendering?)